new/LIT6166MiraclesJohnBaptist.xml

```diff
@@ -53,10 +55,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                  ዓ<supplied reason="omitted">ለም</supplied>፡ አ<supplied reason="omitted">ሜን</supplied>፡
+                  ወነበረ፡ ፩ብእሲ፡ ዘስሙ፡ ሀብተ፡ ኢየሱስ፡ ወውእቱ፡ ብእሲ፡ ፈራሄ፡ እግዚአብሔር፡ ወባዕል፡ ፈድፋደ፨ ወይሁብ፡ ምጽዋተ፡ ለነዳያን፡ ወለመስኪናን፡
+
+                  </ab>
+              </div>
+              <div type="textpart" subtype="chapter" xml:id="Miracle9">
+                <label>Miracle 9</label>
+                <ab>
+                  <pb n="85ra"></pb>
+                  ፱ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ ወበረከቱ፡
+                  የሃሉ፡ ምስለ፡ አግብርቲሁ፡ ወልደ፡ ኪዳን፡ ወለተ፡ ግብርኤል፡ ለ፡ ዓ፡ አ፨
+                  ወነበረ፡ ፩ቀሲስ፡ ዘመጽአ፡ <sic>ተሰዲዮ፡</sic> በረኃብ፡ ወኃደረ፡ ውስተ፡ ብሒር፡
+                  ዘቤተ፡ ቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ወይቤልዎ፡ ካህናት፡ ነዓ፡ ከመ፡ ትርድአነ፡ ለቅኔ፡ ቤተ፡ ክርስቲያን፨
+                  ወይቤሎሙ፡ በትምይንት፡ ኦሆ
+
+                  </ab>
+              </div>
+
+
+<div type="textpart" subtype="chapter" xml:id="Miracle10">
+  <label>Miracle 10</label>
+  <ab>
+    <pb n="85vb"></pb>
+    ፲ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ ጸሎቱ፡ ወበረከቱ፡
+    የሃሉ፡ ምስለ፡ አግብርቲሁ፡ ወልደ፡ ኪዳን፡ ወለተ፡ ሥላሴ፡ ለ፡ ዓ፡ አ።
+    አመ፡ ፪ለሰኔ፡ አስታርእየ፡ ሥጋሁ፡ ለቅዱስ፡ ዮሐንስ፡ መጥምቅ፡
+    ወሥጋሁ፡ ለቅዱስ፡ ኤልሳዕ፡ ረድኡ፡ ለኤልያስ፡ ቀናዒ፨
+    </ab>
+
+<div type="textpart" subtype="chapter" xml:id="Miracle11">
+  <label>Miracle 11</label>
+  <ab>
+    <pb n="87ra"></pb>
+    ፲፩ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ አግብርቲሁ፡
+    አቡነ፡ ወልደ፡ ኪዳን፡ ወለተ፡ ግብርኤል፡ ለዓለመ፡ ዓለም፡ አሜን፨
+    ወሀለወት፡ አሐቲ፡ በዓት፡ ዘትነብር፡ ማዕከለ፡ ፈለገ፡ ውሒዝ፡ እንተ፡ አውሐዘ፡ ቅዱስ፡ ዮሐንስ፡ በኃይለ፡ ሥልጣኑ፨
+</ab>
+</div>
+
+<div type="textpart" subtype="chapter" xml:id="Miracle12">
+  <label>Miracle 12</label>
+  <ab>
+    <pb n="87va"></pb>
+    ፲፪ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ ጸሎቱ፡ የሃሉ፡ ምስለ፡ አግብርቲሁ፡ አቡነ፡ ገብረ፡ ዮሐንስ፡ ወለተ፡ ግብርኤል፡ ለዓለመ፡ ዓለም፡ አሜን፨
+ወሀሎ፡ ፩ብእሲ፡ ዘሥሉጥ፡ ላዕለ፡ ምድረ፡ ኅርጉድ፨ ወሰምዓ፡ ዜና፡ ውእቱ፡ በፅንዓ፡ ኃይሉ፡
+</ab>
+</div>
+<div type="textpart" subtype="chapter" xml:id="Miracle13">
+  <label>Miracle 13</label>
+  <ab>
+    <pb n="88ra"></pb>
+  ፲፫።ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡
+  አግብርቲሁ፡ አቡነ፡ ገብረ፡ ዮሐንስ፡ ወለተ፡ ኢየሱስ፡ ለዓለመ፡ ዓለ፡ አ።<pb n="88rb"></pb> ወካዕበ፡ ስምዑ፡ ንነግረክሙ፡ ዘንተ፡ ተአምረ፡
+  <supplied reason="omitted" resp="DN"/><pb n="88va"></pb> ተብህለ፡ ከመ፡ ሀሎ፡ ብእሲ፡ መኰንነ፡ ቡር፡ ዘይትመን፡ በጽንዑ፡ ከመ፡ አጋግ፡ አማሌቃዊ፨
+  ወአሐተ፡ ዕለተ፡ አስተጋብአ፡ ሠራዊቶ፡ ወሖረ፡ ብትዕቢት፡ ወበተዝኅሮ፡ ከመ፡ ይጽብኦሙ፡ ለሰብአ፡ ሀገሩ፡ ለቅዱስ፡ ዮሐንስ፡ መጥምቅ፨ ወሶበ፡ መጽአ፡ ሰምዑ፡ ገብሩ፡ ምህልላ፡ በትጋህ፡
+</ab>
+</div>
+              </div>
+              </div>
             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESmy018"/>
                     <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESmy023"/>
+                    <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESehpp007"/>
+                    <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESbgy012"/>
                 </listRelation>
```

Comment:
if there is another text/collection of miracles here you need a relation to state the differentFrom. 6165? nothing there...

****
If not already present, you also need, to describe the manuscripts, the overarching ID for the Hagiographic Dossier with Gadl, Miracles, Malk, etc. (cp. LIT1118DossierAntony, LIT5877DossierParaqlitos, etc.)

Reply:
Sorry, I can creat "Dossier" but I do not want to have it in the catalogue. The idea of "Hagiographic dossier" is not suitable for a catalogue. In presentation of the content it creats a level that does not exist in the reality (though I can imagine such an entry as point of reference to all text dedicated to the saint, unless the record for a saint is not sutisfactory (why?)). "H dossier" is an overarching comprehensive term that is used in the hagiographic research to refer summarily to all texts related to the saint, major and minor (and their versions). Dossier includes dozens of texts, big and small, independent of parts of other compositions. Normaly dossier has by far more texts than one manuscript contain.

Reply:
of course, msItems show which actual parts of a dossier are present in a manuscript, yet it is important to show if a manuscript has e.g. only Gadl, or only Miracles, or a set of texts dedicated to one saint, also to refer to a MS. 

also when we refer to a group ID for the miracles of Mary we do not imply that ALL miracles are in the MS

Reply:
"it is important to show if a manuscript has e.g. only Gadl, or only Miracles, or a set of texts dedicated to one saint," - but what for do I prepare a description with works listed as items? If one wants to show what parts of a dossier are included in the manuscript (which is self evident in the description, in my opinion), one needs to carry out a large work to define the dossier, which I do not want to enter now for John the Baptist. And, again, this is not applicable for a catalogue. "Miracles of Mary" is a different matter.